hbase-rest/src/main/java/org/apache/hadoop/hbase/rest/model/NamespacesInstanceModel.java

```diff
@@ -166,4 +167,17 @@ public ProtobufMessageHandler getObjectFromMessage(byte[] message) throws IOExce
     }
     return this;
   }
+
+  public static void main(String[] args) {
```

Comment:
Can we do this as a unit test instead of a main method someone has to run?

Reply:
This should actually be removed, just had temporarily added this in order to have a proper json string generated, to perform manual tests using an http client.